Sources/SwiftParserDiagnostics/ParserDiagnosticMessages.swift

```diff
@@ -542,3 +548,24 @@ public struct ReplaceTokensFixIt: ParserFixIt {
     "replace \(nodesDescription(replaceTokens, format: false)) by '\(replacement.text)'"
   }
 }
+
+public struct ReplacePoundElseSpaceIfTypo: ParserFixIt {
```

Comment:
Fix-Its in SwiftParser describe the action they are performing (for example see how the messages above say e.g. `insert newline`). So this message should be `replace '#else if' by '#elseif'`. We have that exact message above, so you should be able to reuse that exact message in the diagnostic generator by calling the initializer of `ReplaceTokensFixIt`. You can search the diagnostic generator for examples how to do that.